tests/functional/object/mcg/test_write_to_bucket.py

```diff
@@ -456,3 +459,43 @@ def test_content_encoding_with_write(
         logger.info(
             "Put object operation is preserving ContentEncoding as a object metadata"
         )
+
+    @tier2
+    @bugzilla("2259189")
+    @bugzilla("2264480")
+    @pytest.mark.polarion_id("OCS-5773")
+    def test_nb_db_activity_logs_on_io(
+        self, bucket_factory, awscli_pod_session, mcg_obj
+    ):
+
+        """
+        This test checks if the activity logs are being logged
+        in the activitylogs table for every object upload and
+        deletion when noobaa log is set to default
+
+        """
+
+        bucket = bucket_factory()[0]
+        full_objectpath = f"s3://{bucket.name}"
+        logger.info("successfully created bucket")
+
+        sync_object_directory(
+            awscli_pod_session, AWSCLI_TEST_OBJ_DIR, full_objectpath, mcg_obj
+        )
+        logger.info("uploaded objects to the bucket")
+
```

Comment:
1. Can we expect the activity logs to be immediately available after the I.O operations? The test might pass when it shouldn't due to a race condition, if the logs take a bit to get written in the DB. If that might be the case, please add a sleep period in between.

2. Other existing or potential future tests might do I.O with elevated noobaa logs. If such a test will run before this one, the test might fail when it shouldn't due to the old logs. You can prevent this by filtering out logs that don't mention the name of the bucket that's used here:
```
nb_activitylogs = [log for log in nb_activitylogs if bucket.name in log]
```

Reply:
Thanks for addressing my comments in https://github.com/red-hat-storage/ocs-ci/pull/9720/commits/d252c8df12b5dd6d77acf28d63253e92a1905fa3